cmd/proxy/main.go

```diff
@@ -58,12 +87,44 @@ func main() {
 
 	switch {
 	case maxConcurrency == 0:
-		log.Fatal("--max-concurrency flag is required")
+		log.Fatal("-max-concurrency flag is required")
 	case maxQueueLength == 0:
-		maxQueueLength = maxConcurrency * 10
+		log.Fatal("-max-queue-length flag is required")
+	case clusterConfigPath == "":
```

Comment:
is the full config required?

Reply:
I think it's required for re-generating the user ID and for getting the telemetry.